ui/app/AppLayouts/Onboarding/popups/BeforeGetStartedModal.qml

```diff
@@ -28,94 +30,83 @@ StatusDialog {
+                }
             }
+        }
+    }
 
-            StatusCheckBox {
-                id: termsOfUse
-                objectName: "termsOfUseCheckBox"
-                width: parent.width
-                font.pixelSize: 15
-
-                contentItem: Row {
-                    spacing: 4
-                    leftPadding: termsOfUse.indicator.width + termsOfUse.spacing
-
-                    StatusBaseText {
-                        text: qsTr("I accept Status")
-                        font.pixelSize: 15
-                    }
-
-                    StatusBaseText {
-                        objectName: "termsOfUseLink"
-                        text: qsTr("Terms of Use")
-                        color: Theme.palette.primaryColor1
-                        font.pixelSize: 15
-                        font.weight: Font.Medium
-
-                        MouseArea {
-                            anchors.fill: parent
-                            cursorShape: Qt.PointingHandCursor
-                            hoverEnabled: true
-                            onEntered: {
-                                parent.font.underline = true
-                            }
-                            onExited: {
-                                parent.font.underline = false
-                            }
-                            onClicked: {
-                                Qt.openUrlExternally("https://status.im/terms-of-use/")
-                            }
-                        }
-                    }
-
-                    StatusBaseText {
-                        text: "&"
-                        font.pixelSize: 15
-                    }
-
-                    StatusBaseText {
-                        objectName: "privacyPolicyLink"
-                        text: qsTr("Privacy Policy")
-                        color: Theme.palette.primaryColor1
-                        font.pixelSize: 15
-                        font.weight: Font.Medium
-
-                        MouseArea {
-                            anchors.fill: parent
-                            cursorShape: Qt.PointingHandCursor
-                            hoverEnabled: true
-                            onEntered: {
-                                parent.font.underline = true
-                            }
-                            onExited: {
-                                parent.font.underline = false
-                            }
-                            onClicked: {
-                                Qt.openUrlExternally("https://status.im/privacy-policy/")
-                            }
-                        }
-                    }
+    Component {
+        id: detailsPopup
+        StatusDialog {
+            required property string textFile
+            width: 600
+            padding: 0
+            destroyOnClose: true
+            standardButtons: Dialog.Ok
+            anchors.centerIn: parent
```

Comment:
Do we really need this line?

Reply:
And since I realized we have no `Global` at this stage, I had to change the impl a bit (from a `Component` to a real `StatusDialog` opened and closed manually)